homework/Bladyko Danila/HomeWork_03/by/jrr/src/by/jrr/tests/UserServiceTest.java

```diff
@@ -0,0 +1,59 @@
+package by.jrr.tests;

```

Comment:
this test won't work. Ask colleagues for help
Create a new one test using best practices that we discussed